crawler/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta encoding='utf-8'/>
+    <title>Headless detector</title>
+  </head>
+
+  <body>
```

Comment:
Can you please add breadcrumbs and a description like we have on other pages?

<img width="758" alt="header" src="https://user-images.githubusercontent.com/985504/107066824-bca0fd80-67de-11eb-83e0-5be57c19b3db.png">

Sth like "This page collects various tests for detecting headless browsers."